### PR TITLE
{RDBMS} Add rdbms code owner

### PR DESCRIPTION
**Pull request by evelyn-ys**
## Description ##
We may need service team member as rdbms code owner to ensure the code quality as we CLI team members may not know exactly and explictly how everything works.
For example, network migration need service team's review: #16350 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -33,7 +33,7 @@
 /src/azure-cli/azure/cli/command_modules/privatedns/ @jsntcy @haroldrandom @kairu-ms @msyyc
 /src/azure-cli/azure/cli/command_modules/policyinsights/ @cheggert
 /src/azure-cli/azure/cli/command_modules/profile/ @jiasli @evelyn-ys @fengzhou-msft
-/src/azure-cli/azure/cli/command_modules/rdbms/ @Juliehzl @evelyn-ys
+/src/azure-cli/azure/cli/command_modules/rdbms/ @arde0708 @Juliehzl @evelyn-ys
```
Comment (jiasli) [inline .github/CODEOWNERS:36]: 👍 Good to have a member from service team to ensure the code quality.

**Review by jiasli — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (jiasli) [reply .github/CODEOWNERS:36]: You may link to the original issue/PR to make the purpose of this PR clearer.

Comment (yonzhan) [reply .github/CODEOWNERS:36]: @evelyn-ys nice!

**Comment by yonzhan** [comment]
RDBMS

**Review by yonzhan — APPROVED** [review]
LGTM

**Pull request merged**